templates/fine-tune-llm_v2/training_configs/custom/meta-llama/Meta-Llama-3-70B/lora/8xA100-80G-4k.yaml

```diff
@@ -39,3 +39,22 @@ deepspeed:
 # worker.
 worker_resources:
   accelerator_type:A100-80G: 0.001
+
+# Lora configuration
```

Comment:
oh we didn't have lora config :) ? shit!!

Reply:
yeah lol i spent 30 mins trying to figure out why it wasn't running when i was testing liger before sumanth pointed out lora was missing